### PR TITLE
Copy edit CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Lupie**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-This doc is intended for contributors to UberFx (hopefully that's you!)
+This doc is intended for contributors to UberFx (hopefully that's you!).
 
-## Development Environment
+## Development environment
 
-* Go. Install on OS X with `brew install go`. Make sure `go version` returns at
+* **Go**. Install on OS X with `brew install go`. Make sure `go version` returns at
   least `1.7` since we're going to be using 1.7+ features like subtests.
```
Comment (glibsm) [inline CONTRIBUTING.md]: 1.8+ now

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,24 +25,25 @@ cd $GOPATH/src/go.uber.org/fx
...
+## License headers
 
-This project is Open Source Software, and requires a header at the beginning of
-all source files. This is enforced by commit hooks and TravisCI.
+This project is open source software and requires a header at the beginning of
+every source file. This is enforced by commit hooks and TravisCI.
 
-To add licence headers, use
+To add license headers, use
 [uber-licence](https://github.com/uber/uber-licence):
```
Comment (glibsm) [inline CONTRIBUTING.md]: This is out of date, we don't use the npm version anymore.
We now use `go.uber.org/tools/update-license` and it comes as part of `make dependencies`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,24 +25,25 @@ cd $GOPATH/src/go.uber.org/fx
...
 
 ```lang=bash
 make add-uber-licence
 ```
+Note that `uber-licence` is spelled with two c's for historical reasons.
```
Comment (glibsm) [inline CONTRIBUTING.md]: can also be removed - spelling is now ok

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -76,15 +77,15 @@ make test
 
 ### Docker
```
Comment (glibsm) [inline CONTRIBUTING.md]: we do not run on docker for the time being, can nuke this whole section now

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
 # Developing UberFx
```
Comment (breerly) [inline CONTRIBUTING.md]: @Lupie - we need to call the project the same thing everywhere: Fx

Can you make a pass at all the docs in this repo to make sure we aren't calling it UberFx anymore?

**Comment by Lupie** [comment]
Oh, that's news to me. Yep, no prob.

**Pull request merged**